app/component/CarLeg.js

```diff
@@ -19,6 +19,8 @@ function CarLeg(props, context) {
   const firstLegClassName = props.index === 0 ? 'start' : '';
   const modeClassName = 'car';
 
+  const [address, place] = props.leg.from.name.split(/, (.+)/); // Splits the name-string to two parts from the first occurance of ', '
```

Comment:
I'm not sure if there is already identical function somewhere but seems like this could be useful elsewhere as well and should perhaps be put into otpStrings.js or some other util.